Sources/NIOConcurrencyHelpers/atomics.swift

```diff
@@ -473,6 +545,17 @@ public final class AtomicBox<T: AnyObject> {
     ///
     /// - Parameter value: The new value to set the object to.
     public func store(_ value: T) -> Void {
```

Comment:
This should probably also have the warning about using a CAS loop

Reply:
@drexin that one doesn't actually have one though, right?

Reply:
@weissi It calls `exchange`, which does.

Reply:
Oh, I see... it calls it on `storage` directly. Ignore what I said.